support/build/Makefile.rules

```diff
@@ -618,16 +618,17 @@ endef
 define buildrule_S_lds =
 $(4): $(2) | preprocess
 	$(call build_cmd_fixdep,LDS,$(1),$(4),\
-		$(AS)  -E -P -x assembler-with-cpp $$(COMPFLAGS) $$(COMPFLAGS-y) \
-		       $$(ASINCLUDES) $$(ASINCLUDES-y) \
-		       $$($(call vprefix_lib,$(1),ASINCLUDES)) $$($(call vprefix_lib,$(1),ASINCLUDES-y)) \
-		       $$($(call vprefix_src,$(1),$(2),$(3),INCLUDES)) $$($(call vprefix_src,$(1),$(2),$(3),INCLUDES-y)) \
-		       $$(ARCHFLAGS) $$(ARCHFLAGS-y) \
-		       $$(ASFLAGS) $$(ASFLAGS-y) $$(ASFLAGS_EXTRA) \
-		       $$($(call vprefix_lib,$(1),ASFLAGS)) $$($(call vprefix_lib,$(1),ASFLAGS-y)) \
-		       $$($(call vprefix_src,$(1),$(2),$(3),FLAGS)) $$($(call vprefix_src,$(1),$(2),$(3),FLAGS-y)) \
+		$(AS)  -E -P -x assembler-with-cpp \
 		       $(5) \
-		       $(2) -o $(4) $(call depflags,$(4))
+		       $(2) -o $(4) $(call depflags,$(4)), \
```

Comment:
Same question about the `,`. I don't think we need to reformat all of this.

Reply:
See https://github.com/unikraft/unikraft/pull/619/files#r1031575279